cluster-sync/k8s-1.15.1/provider.sh

```diff
@@ -12,8 +12,32 @@ fi
 
 if [[ $KUBEVIRT_PROVIDER_EXTRA_ARGS =~ "--enable-ceph" ]]; then
   echo "Switching default storage class to ceph"
-  #Switch the default storage class to ceph
+  # Switch the default storage class to ceph
   _kubectl patch storageclass local -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
-  _kubectl patch storageclass csi-rbd -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+  # Make sure the SC is configured to use xfs instead of ext4
+  _kubectl delete storageclass csi-rbd
+  cat <<EOF | _kubectl apply -f -
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
```

Comment:
I think this is fine but wonder if if would be better to do in kubevirtci?

Reply:
I would actually prefer that kubevirtci doesn't install any storage providers at all, maybe provide some scripts to active the ones you want. But at this point I would like to minimize the storage modifications in kubevirtci.